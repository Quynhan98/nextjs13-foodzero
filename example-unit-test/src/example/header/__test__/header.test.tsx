import * as renderer from 'react-test-renderer';
import { Header } from '..';

describe('test header', () => {
  test('snap', () => {
    const component = renderer.create(<Header />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
