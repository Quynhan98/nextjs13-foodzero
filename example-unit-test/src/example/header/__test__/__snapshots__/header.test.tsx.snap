// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header snap 1`] = `
<header>
  <h2
    className="logo"
  >
    <img
      alt=""
      src="IMAGE_MOCK"
    />
  </h2>
</header>
`;
